package.json

```diff
@@ -48,11 +49,12 @@
     "cover": "istanbul cover node_modules/.bin/_mocha -- -u exports --compilers js:babel/register -R spec",
     "coveralls": "NODE_ENV=test ./node_modules/.bin/istanbul cover ./node_modules/mocha/bin/_mocha --report lcovonly -- -u exports --compilers js:babel/register -R spec && cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js",
```

Comment:
@bruderstein changed in https://github.com/JedWatson/react-select/commit/65358b5f8a477d68f21bc83485d7975785c5b162